sources/packages/backend/apps/db-migrations/src/sql/Types/Rollback-add-stop-part-time-bc-funding-restriction-action-type.sql

```diff
@@ -0,0 +1,30 @@
+--Removing the value Application from sims.note_types for the rollback.
```

Comment:
I believe this comment needs to be updated.

Reply:
Nice catch!